cmd/testrunner.go

```diff
@@ -156,15 +164,27 @@ func testTypeCommandActionFactory(runner testrunner.TestRunner) cobraext.Command
 			return fmt.Errorf("cannot determine if package has data streams: %w", err)
 		}
 
+		configFileFlag, _ := cmd.Flags().GetString(cobraext.ConfigFileFlagName)
+		if configFileFlag != "" && !filepath.IsAbs(configFileFlag) {
+			return fmt.Errorf("config file path must be an absolute path: %s", configFileFlag)
+		}
```

Comment:
We can obtain here the absolute path if we need it, right? This improves UX.
```suggestion
		if configFileFlag != ""  {
			absPath, err := filepath.Abs(configFileFlag)
			if err != nil {
				return fmt.Errorf("cannot obtain the absolute path for config file path: %s", configFileFlag)
			}
			configFileFlag = absPath
		}
```